src/rust-crypto/rust-crypto.ts

```diff
@@ -43,12 +43,13 @@ import {
 import { deviceKeysToDeviceMap, rustDeviceToJsDevice } from "./device-converter";
 import { IDownloadKeyResult, IQueryKeysRequest } from "../client";
 import { Device, DeviceMap } from "../models/device";
-import { AddSecretStorageKeyOpts, ServerSideSecretStorage } from "../secret-storage";
+import { AddSecretStorageKeyOpts, SECRET_STORAGE_ALGORITHM_V1_AES, ServerSideSecretStorage } from "../secret-storage";
 import { CrossSigningIdentity } from "./CrossSigningIdentity";
 import { secretStorageContainsCrossSigningKeys } from "./secret-storage";
 import { keyFromPassphrase } from "../crypto/key_passphrase";
 import { encodeRecoveryKey } from "../crypto/recoverykey";
 import { crypto } from "../crypto/crypto";
+import { ICryptoCallbacks } from "../crypto";
```

Comment:
I'm trying to move stuff *out* of `src/crypto` rather than use it as-is. Basically I want to be able to delete the entire directory once ER is stable. Could you move `ICryptoCallbacks` into `src/crypto-api.ts` or somewhere like that?

We should probably move `keyFromPassphrase`, `encodeRecoveryKey` and the `crypto` shims at some point too, but that can happen separately.